src/screens/FreeProducts/utils/apiResponseType.tsx

```diff
@@ -0,0 +1,143 @@
+/**
```

Comment:
Do we really need all this interfaces ? 
Why not fetching the data you need in the backend and return a new interface which contains all information you need ?

Reply:
Why not have all this?

Reply:
I do agree that the ideal would be to do all requests on BE and just send the needed information to the frontend like we are doing with the other requests. This way we can avoid making the frontend too bloated. 👍🏽

Reply:
Doing exactly that with the wine ge implementation. Fetching information in the BE and just return the needed information.